x-pack/plugins/security_solution/server/usage/detections/detection_rule_helpers.ts

```diff
@@ -112,6 +140,16 @@ export const updateDetectionRuleUsage = (
 ): DetectionRulesTypeUsage => {
   let updatedUsage = usage;
 
+  const legacyNotificationEnabled =
```

Comment:
Do `detectionRuleMetric.enabled` and `detectionRuleMetric.disabled` refer to the enabled/disabled status of the rule? Would we want to instead check the `muteAll`? Can you have enabled actions on disabled rules?

Reply:
>Do detectionRuleMetric.enabled and detectionRuleMetric.disabled refer to the enabled/disabled status of the rule?

That is correct.

>Would we want to instead check the muteAll?

I do check `muteAll` when it is a non-legacy rule. If it's `muteAll` I don't count the notification as being enabled. I don't check all the individual actions for all their individual `mute` since we don't have that within our UI at the moment fwiw. If it is legacy, even if `no_actions` is selected from the UI I still count it as having a legacy since technically it does and still needs to be migrated.

>Can you have enabled actions on disabled rules?

You can, and that would set in the values of `notifications_disabled` and `legacy_notifications_disabled`. If you want to get the rules without notifications you have to add these fields up and subtract them from the total rules.